File: fibonacci.pl
use 5.010;

my %evolutions = (
    o => 'x',
    x => 'ox',
);

my $families = 'o';

if ( $ARGV[0] !~ /^[1-9][0-9]*$/ or $ARGV[0] > 28 ) {
    warn "Index must be a positive integer between 1 and 28.";
    exit;
}

my @index = (1..$ARGV[0]-1);
for (@index) {
    $families =~ s/([ox])/$evolutions{$1}/g;
}
say $count = () = $families =~ m/./g;

File: fibonacci.php
<?php 
    if ($argv[1] < 1 || $argv[1] > 92) {
        throw new Exception("Index must be a positive integer between 1 and 92.");
    } 
    $previous = 1;
    $current = 1;
    $next = 0;
    for ($iteration = 2; $iteration < $argv[1]; $iteration++) {
        $next = $previous + $current;
        $previous = $current;
        $current = $next;
    }
    echo $current;
    echo "\n";
?>